ballerina/client.bal

```diff
@@ -56,82 +56,126 @@ public isolated client class Client {
         return;
     }
 
+    # Delete a block
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - OK 
+    resource isolated function delete v1/blocks/[string id](DeleteV1BlocksIdHeaders headers = {}) returns record {}|error {
+        string resourcePath = string `/v1/blocks/${getEncodedUri(id)}`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        return self.clientEp->delete(resourcePath, headers = httpHeaders);
+    }
+
+    # Retrieve a block
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - OK 
+    resource isolated function get v1/blocks/[string id](GetV1BlocksIdHeaders headers = {}) returns record {}|error {
+        string resourcePath = string `/v1/blocks/${getEncodedUri(id)}`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        return self.clientEp->get(resourcePath, httpHeaders);
+    }
+
     # Retrieve block children
     #
-    # + id - block children ID
     # + headers - Headers to be sent with the request 
     # + queries - Queries to be sent with the request 
-    # + return - 200 Success - Retrieve block children 
-    resource isolated function get v1/blocks/[string id]/children(RetrieveBlockChildrenHeaders headers = {}, *RetrieveBlockChildrenQueries queries) returns BlockChildrenResponse|error {
+    # + return - OK 
+    resource isolated function get v1/blocks/[string id]/children(GetV1BlocksIdChildrenHeaders headers = {}, *GetV1BlocksIdChildrenQueries queries) returns record {}|error {
```

Comment:
In the generated client, some resource functions now return open records (record {}) instead of specific record types, unlike the previous spec. I've updated an example to illustrate how to include references like:

```yaml
schema:
  $ref: "#/components/schemas/BlockChildrenResponse"
 ```
 
Can you apply this update to the remaining client APIs as well?

Reply:
And add all the types in the components field in the old-openapi spec to the new spec